packages/gatsby-theme-newrelic/src/components/SwiftSearch.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import SiteSearchAPIConnector from '@elastic/search-ui-site-search-connector';
+import {
+  SearchProvider,
+  WithSearch,
+  SearchBox,
+  Results,
+  Paging,
+  PagingInfo,
+} from '@elastic/react-search-ui';
+import ResultView from './ResultView';
+import PagingInfoView from './PagingInfoView';
+import SearchInput from './SearchInput';
+
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+
+// TODO: move styles to emotion/wrapper
+import Icon from './Icon';
+import styles from '../styles/SwiftSearchStyles';
+
+const connector = new SiteSearchAPIConnector({
+  documentType: 'page',
+  engineKey: 'Ad9HfGjDw4GRkcmJjUut',
```

Comment:
Just want to double-check, this key is okay to be public facing?

Reply:
yes! this is a public read-only key meant to be used in client side api calls